### PR TITLE
Fix file addition problem in xcodeproj

### LINKED ISSUE #29: Linking problems with apps split into modules
When an app has multiple targets with the same platform, e.g. an „App“ target an an „AppKit“ target and both include the same framework, then Accio currently adds only a single framework to the group properly. It seems to try to add a second link, too, but fails - Xcode is placing the second under „Reconstructed Links“ (or similar).

A workaround is to link the frameworks manually, Accio correctly handles them on updates from then on. But Accio should link them correctly in the first place.

### PR DESCRIPTION
**Pull request by mrylmz**
Fixes #29.

### CODE REVIEW

File: Package.resolved [hunk 1]
```diff
@@ -65,12 +65,12 @@
         }
       },
       {
-        "package": "xcodeproj",
-        "repositoryURL": "https://github.com/tuist/xcodeproj.git",
+        "package": "XcodeProj",
+        "repositoryURL": "https://github.com/mrylmz/xcodeproj.git",
```
Comment (Jeehut) [inline Package.resolved:69]: This is fine temporarily, but we should have a PR on the project explaining the fix before this can be merged.

**Review by Jeehut — COMMENTED** [review]
Looks good, but please post a PR on XcodeProj and also make sure the tests are passing. Then we can merge.

File: Package.resolved (hunk 1 above)
Comment (mrylmz) [reply Package.resolved:69]: [👍](https://github.com/tuist/xcodeproj/pull/410)

**Comment by Jeehut** [comment]
Great, thanks! 🎉 
Merging.

**Review by Jeehut — APPROVED** [review]

**Pull request merged**